src/postprocess/chi0.jl

```diff
@@ -0,0 +1,66 @@
+using ForwardDiff
+using ProgressMeter
+
+
+"""
+Compute the independent-particle susceptibility. Will blow up for large systems
+"""
+function compute_χ0(ham)
+    # We're after χ0(r,r') such that δρ = int_r' χ0(r,r') δV(r') dr'
+    # where (up to normalizations)
+    # ρ = ∑_nk f(εnk - εF) |ψnk|^2
+    # ∑_nk f(εnk - εF) = N_el
+    # Everything is ∑med on k so we omit it for notational simplicity
+    # δρ = ∑_nk (f'n δεn |ψn|^2 + 2Re fn ψn* δψn - f'n δεF |ψn|^2
+    # Now we use
+    # δεn = <ψn|δV|ψn>
+    # δψn = ∑_{m != n} <ψm|δV|ψn> |ψm> / (εn-εm)
+    # δN_el = 0 = ∑_n f'n (δεn - δεF) => δεF = 1/DOS int δV(r) LDOS(r)dr
```

Comment:
Prepend: "Also the number of electrons `N_el = Σ_n fn` is constant, thus: ...."

Reply:
Also: That `=>` is still not immediately clear like so.

Reply:
Ah now I finally see what you do. Could you add as well the line `LDOS(r) = ∑_n f'n |ψn|^2 => \int LDOS(r) δV(r) dr = ∑_n f'n δεn`